Raspi/Test/MockSensor.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+
+class MockSensor:
+    """ Base class to make a mock sensor that produces random data"""
+
+    def __init__(self, *ave_val, percent_range=0.05, decimals=2):
+        """
+        ave_val:        Either a single val that sets the average value for the
+                        sensor or and array of subvals formatted as
```

Comment:
Is this meant to be: "sensor or **an** array of subvals formatted as"

Reply:
Yes it is meant to be